picky/src/key.rs

```diff
@@ -202,6 +215,22 @@ impl PrivateKey {
         }))
     }
 
+    pub fn from_ec_der<T: ?Sized + AsRef<[u8]>>(der: &T) -> Result<Self, KeyError> {
```

Comment:
Not wrong, but `AsRef` imply `?Sized`
```suggestion
    pub fn from_ec_der<T: AsRef<[u8]>>(der: &T) -> Result<Self, KeyError> {
```

(I see this is probably a copy paste, you can remove `?Sized` above as well)

Reply:
Isn't `?Sized` required to `T` that implements `AsRef<[u8]>`? `AsRef<[u8]>` only applies `?Sized` to the `[u8]`, doesn't it? I can remove `?Sized`, but this will break compilation some picky internal code(not a big deal, I can fix it easily) and people code that uses picky(is it ok?).

Reply:
Oh, you’re right! Let’s leave the `?Sized`